src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -1,27 +1,32 @@
 package com.google.opensesame.servlets;
 
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.Filter;
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 import com.google.gson.Gson;
+import com.google.opensesame.auth.AuthServlet;
+import com.google.opensesame.util.ErrorResponse;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.kohsuke.github.GHMyself;
+import org.kohsuke.github.GitHub;
 
 @WebServlet("/user")
 public class UserServlet extends HttpServlet {
 
+  private String pageToRedirectToIfUserNotAuthenticated = "index.html";
```

Comment:
nit: Consider something a little shorter, this is a bit hard to read imo.

```suggestion
  private String noAuthenticationRedirect = "index.html";
```

Reply:
I'm getting rid of that functionality as a whole to keep the Servlet Restful. Good catch

Reply:
I actually deprecated this functionality in order to keep the Servlet Restful. Good catch!
```suggestion
```